File: specification/specs/cluster/task_management/cancel_tasks/CancelTasksRequest.ts
@rest_spec_name("tasks.cancel")
class CancelTasksRequest extends RequestBase {
  path_parts?: {
    task_id?: string;
  }
  query_parameters?: {
    actions?: string[];
    nodes?: string[];
    parent_task_id?: string;
  }
  body?: {
  }
}

File: output/csharp/Cluster/TaskManagement/CancelTasks/CancelTasksRequest.cs
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Cluster {

	public class CancelTasksRequest : RequestBase {
		
		[DataMember(Name="actions")]
		public List<string> Actions { get; set; }


		[DataMember(Name="nodes")]
		public List<string> Nodes { get; set; }


		[DataMember(Name="parent_task_id")]
		public string ParentTaskId { get; set; }

	}
}
